public/javascripts/Gallery/src/modal/Modal.js

```diff
@@ -24,6 +24,8 @@ function Modal(uiModal) {
                 $('.gallery-modal').attr('style', 'display: flex');
                 $('.grid-container').css("grid-template-columns", "1fr 5fr");
 
+                self.open = true;
```

Comment:
And if we can include set this in `openModal()`, we shouldn't need to do it here. But maybe you're doing it before `openModal()` is called for a reason?